bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -30,11 +30,11 @@ export default function userLeaving(credentials, userId) {
       'user-not-found', `You need a valid user to be able to toggle audio`);
   }
 
-  if (User.user.connection_status === OFFLINE_CONNECTION_STATUS) {
+  if (User.connection_status === OFFLINE_CONNECTION_STATUS) {
```

Comment:
Are you sure this will work? The `connections_status` is saved inside the nested user object.

```
UserSchema {
  userID: xxx,
  ...
  user: {
    connections_status: 'online',
    ...
  }
}
```